app/models/models.py

```diff
@@ -1,9 +1,69 @@
-from sqlalchemy.schema import Column
-from sqlalchemy.types import String
-import bigfastapi.db.database as db
+import datetime
 from uuid import uuid4
 
+import bigfastapi.db.database as db
+from sqlalchemy import ForeignKey, Integer
+from sqlalchemy.schema import Column
+from sqlalchemy.types import DateTime, String
+
+
 class Test(db.Base):
     __tablename__ = "test"
     id = Column(String(255), primary_key=True, index=True, default=uuid4().hex)
     text = Column(String(255), index=True)
+
+
+class Village(db.Base):
+    __tablename__ = "villages"
+    id = Column(String(255), primary_key=True, index=True, default=uuid4().hex)
+    name = Column(String(255), index=True)
+    loation = Column(String(255), ForeignKey("locations.id"))
+    contributed_by = Column(String(255), ForeignKey("users.id"))
+
+
+class Voter(db.Base):
+    __tablename__ = "voters"
+    id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
+    village = Column(String(255), ForeignKey("villages.id"))
+    name = Column(String(255), index=True)
+    contact = Column(String(255), ForeignKey("contacts.id"))
+    notes = Column(String(255), index=True)
+    importance = Column(String(255), index=True)
+    date_delivered = Column(DateTime, default=datetime.datetime.now)
```

Comment:
I believe the `default=datetime.datetime.now` should be `default=datetime.datetime.now()`